indico/modules/rb/schemas.py

```diff
@@ -401,6 +401,10 @@ def _check_room_name_format_placeholders(self, room_name_format, **kwargs):
             # validated client-side, no i18n needed
             raise ValidationError('Missing placeholders: {}'.format(', '.join(missing)))
 
+        placeholders = get_format_placeholders(room_name_format)
```

Comment:
Maybe we can catch ValueError here and turn it into something like "Invalid format string" (as a ValidationError, this one with i18n)? This happens e.g. if you put just `{` at the end of the string.